presto-parquet/.gitignore

```diff
@@ -0,0 +1,2 @@
+src/main/java/ParquetFlatColumnReaderGenerator.tdd
```

Comment:
Can you please explain why we need to re-generate the readers? It'll be nice to add a commit message to explain it.

Reply:
Previously during the build of this module we would always generate them. A recent PR committed the generated files (#23699) by copying them into the main source tree but removed the code generation mechanism. This PR made use of the generation mechanism to create the batch reader for the UUID type. I also think it's fine to keep the generation disabled behind a profile if we want to keep the files checked into the source tree. But we should entirely remove the mechanism which created the files in the first place. 

I will add a comment to the commit message to clarify this.